### PR TITLE
DOC: Update section for SQuirrel SQL 4.0.0

### PR DESCRIPTION
**Pull request by costin**
The just released SQuirrel SQL 4.0.0 provides an Elasticsearch driver
definition out of the box; update the documentation to reflect that.

### CODE REVIEW

**Review by matriv — APPROVED** [review]
LGTM

File: docs/reference/sql/endpoints/client-apps/squirrel.asciidoc [hunk 1]
```diff
@@ -20,22 +21,21 @@ To add the {es} JDBC driver, use *Windows* > *View Drivers* menu (or Ctrl+Shift+
...
 
-The driver should now appear in the list:
+The driver should now appear in the list with a blue check mark next to its name:
 
 image:images/sql/client-apps/squirell-4-driver-list.png[]
 
 ==== Add an alias for {es}
 
-Add a new connection or in SQuirelL terminology an _alias_ using the new driver. To do so, select the *Aliases* panel on the left and click the `+` sign:
+Add a new connection or in SQuirrelL terminology an _alias_ using the *Elasticsearch* driver. To do so, select the *Aliases* panel on the left and click the `+` sign:
```
Comment (astefan) [inline docs/reference/sql/endpoints/client-apps/squirrel.asciidoc]: Typo: single `L`.

File: docs/reference/sql/endpoints/client-apps/squirrel.asciidoc [hunk 2]
```diff
@@ -47,6 +47,6 @@ The setup is completed. Double check it by clicking on *Test Connection*.
 
 ==== Execute SQL queries
 
-The connection should open automatically (if it has been created before simply click on *Connect* in the *Alias* panel). SQuirelL SQL can now issue SQL commands to {es}:
+The connection should open automatically (if it has been created before simply click on *Connect* in the *Alias* panel). SQuirrelL SQL can now issue SQL commands to {es}:
```
Comment (astefan) [inline docs/reference/sql/endpoints/client-apps/squirrel.asciidoc]: Same typo with doubled `L`.

**Review by astefan — APPROVED** [review]
LGTM. Left two minor comments about small typos.

**Pull request merged**